stock_move_backdating/__openerp__.py

```diff
@@ -19,23 +19,19 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
+
+
 {
-    "name": "Stock Move Backdating",
-    "version": "1.0",
-    'author': ['Marco Dieckhoff, BREMSKERL', 'Agile Business Group'],
-    "category": "Stock Logistics",
+    'name': 'Stock Move Backdating',
+    'version': '8.0.0.1.0',
```

Comment:
here seems be possible to use also 8.0.0.1.0
https://github.com/archetipo/stock-logistics-workflow/blob/porting_stock_move_backdating/README.md#available-addons